app/models/provider.rb

```diff
@@ -60,6 +60,9 @@ class Provider < ApplicationRecord
   has_many :users, through: :organisations
 
   has_many :sites
+
+  # NOTE: To be removed
+  #       START
```

Comment:
Can we know why?

Reply:
The the provider enrichment block on the provider...

https://trello.com/c/brUzXXDk/2383-remove-providerenrichment#

Reply:
Next time let's put this into the code so it's easier to understand for our tomorrow-selves. :)